msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -283,59 +283,103 @@ To build am API-based message extension using Visual Studio Code, follow these s
 
  A new browser window with Teams web client opens. You can add your app to Teams.
 
-<!--# [Visual Studio](#tab/visual-studio)
+# [Visual Studio](#tab/visual-studio)
 
-Before you get started, ensure that you install the Visual Studio Enterprise 2022 Preview version 17.9.0 Preview 1.0, and install the **Microsoft Teams development tools** under **ASP.NET and web development** workload.
+Before you get started, ensure that you install the following tools to build and deploy your message extension:
+
+* Install the Visual Studio Enterprise 2022 Preview version 17.9.0 Preview 1.0 and install the **Microsoft Teams development tools** under **ASP.NET and web development** workload.
+* Ensure that the Copilot feature flag is enabled. To enable the feature flag, follow these steps:
+  1. Open **Visual Studio**.
+  1. Go to **Tools** > **Options**.
+  1. Enter **Teams Toolkit** in the **Search Settings** search box.
+  1. Under **Preview Features**, select the **Teams Toolkit: Develop Copilot Plugin** checkbox.
```

Comment:
I think this is not required for API ME in VS

Reply:
removed.

Reply:
tested by removing the copilot feature flag. Confirmed that the option is not required.